pkg/controller/controller.go

```diff
@@ -171,34 +175,39 @@ func (c *controller) reconcileBroker(broker *v1alpha1.Broker) {
 		glog.V(4).Infof("Adding/Updating Broker %v", broker.Name)
 		brokerCatalog, err := brokerClient.GetCatalog()
 		if err != nil {
-			glog.Errorf("Error getting broker catalog for broker %v: %v", broker.Name, err)
-			c.updateBrokerReadyCondition(broker, v1alpha1.ConditionFalse, errorFetchingCatalogReason, errorFetchingCatalogMessage)
-
+			s := fmt.Sprintf("Error getting broker catalog for broker %q: %s", broker.Name, err)
+			glog.Info(s)
```

Comment:
Why is this an error in the service catlog? What does the operator of the service catalog need to do to remedy it?

This looks to me more like a user error. The url was wrong, the user/pass was wrong, the catalog was improperly formed. I don't see how telling the operator of the service catalog itself has any duty to do anything, or even can necessarily do anything.

This is a warning at best, and really only if it's happening a lot, as some sort of network misconfiguration or ddos.